rclcpp/CMakeLists.txt

```diff
@@ -127,6 +127,32 @@ if(BUILD_TESTING)
     )
     target_link_libraries(test_find_weak_nodes ${PROJECT_NAME})
   endif()
+
+  get_default_rmw_implementation(default_rmw)
+  find_package(${default_rmw} REQUIRED)
```

Comment:
is it a requirement that this test `find_package`s and uses another rmw than all the other tests of this package?